mlir/lib/Dialect/Rock/Generator/Conv2dGenerator.cpp

```diff
@@ -556,6 +568,7 @@ LogicalResult Conv2dGenerator::parseConvConfig(const char *arguments) {
   strToInt("padding_h", config.paddingHeightRight);
   strToInt("padding_w", config.paddingWidthLeft);
   strToInt("padding_w", config.paddingWidthRight);
+  config.outputDataTypeStr = canonicalizeDataType(argMap["out_type"]);
```

Comment:
It's more aesthetic, but could you move this up after the `config.dataTypeStr`? It looks out of place here